src/esp/gfx/SkinData.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
```

Comment:
Instead of pragma the Facebook design guide says we should use header guards. 

Also needs the copyright